pkg/node/manager/manager.go

```diff
@@ -477,6 +478,10 @@ func (m *manager) NodeUpdated(n nodeTypes.Node) {
 			continue
 		}
 
+		if addr := address.ToString(); address.Type == addressing.NodeCiliumInternalIP && m.conf.IsLocalRouterIP(addr) {
+			continue
```

Comment:
Actually, I think we want 

```suggestion
			dpUpdate = false
```

here instead to not break other behavior like inserting the IP into the ipcache.

Reply:
Did you mean `dpUpdate = true`? Anyways, I think we have two options here with the lowest chance to break other stuff:
a) Skip setting `dpUpdate = false` when IP matches criteria [1]
b) Skip processing when IP matches criteria

Unfortunately I couldn't track down what happens if there is `ipcache` or not for `LocalRouterIP`.

[1] criteria are: is `NodeCiliumInternalIP`, and matches either `LocalRouterIP`

Reply:
We still want the ipcache entry for that IP.